src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Stream;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int SECOND_ELEMENT = 1;
+    private static final String TXT_SUPPLY = "supply";
+    private static final String TXT_BUY = "buy";
+    private static final String TXT_RESULT = "result";
+    private static final String TXT_SEPARATOR = ",";
+
+    private void appendStringBuilder(StringBuilder stringBuilder,
+                                     String text, int number) {
+        stringBuilder.append(text).append(number).append(System.lineSeparator());
+    }
 
+    private List<String> readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        Stream<String> lines = null;
+        try {
+            lines = Files.lines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        return lines.toList();
+    }
+
+    private int addElementIfContains(String line, String isContained,
+                                      String[] fromArray, int element) {
+        if (line.contains(isContained)) {
+            int value = Integer.parseInt(fromArray[element]);
+            return value;
+        }
```

Comment:
The method `addElementIfContains` is not clear and it's doing more than one thing which contradicts the Single Responsibility Principle. It's better to split this method into two: one for checking if a line contains a certain string, and another for parsing an integer from a string array.